firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionViewModel.swift

```diff
@@ -0,0 +1,144 @@
+    var onToggleSiteSafelistStatus: (() -> Void)?
+    var heroImage: UIImage?
+
+    // MARK: - Constants
+    let contentBlockerStatus: BlockerStatus
+    let contentBlockerStats: TPPageStats?
+    var certificates = [Certificate]()
+    let url: URL
+    let displayTitle: String
+    let connectionSecure: Bool
+    let globalETPIsEnabled: Bool
+
+    let clearCookiesButtonTitle: String = .Menu.EnhancedTrackingProtection.clearDataButtonTitle
+    let clearCookiesButtonA11yId: String = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.clearCookiesButton
+
+    let settingsButtonTitle: String = .Menu.EnhancedTrackingProtection.privacySettingsTitle
+
+    let clearCookiesAlertTitle: String = .Menu.EnhancedTrackingProtection.clearDataAlertTitle
+    let clearCookiesAlertText: String = .Menu.EnhancedTrackingProtection.clearDataAlertText
+    let clearCookiesAlertButton: String = .Menu.EnhancedTrackingProtection.clearDataAlertButton
+    let clearCookiesAlertCancelButton: String = .Menu.EnhancedTrackingProtection.clearDataAlertCancelButton
+
+    // MARK: Accessibility Identifiers
+    let foxImageA11yId: String = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.foxImage
+    let shieldImageA11yId: String = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.shieldImage
+    let lockImageA11yId: String = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.lockImage
+    let arrowImageA11yId: String = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.arrowImage
+
+    let settingsA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.trackingProtectionSettingsButton
+    let domainLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.domainLabel
+    let domainHeaderLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.domainHeaderLabel
+    let statusTitleLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.statusTitleLabel
+    let statusBodyLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.statusBodyLabel
+    let trackersBlockedLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.trackersBlockedLabel
+    let securityStatusLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.securityStatusLabel
+    let toggleViewTitleLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.toggleViewTitleLabel
+    let toggleViewBodyLabelA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.toggleViewBodyLabel
+    let closeButtonA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.closeButton
+    let faviconImageA11yId = AccessibilityIdentifiers.EnhancedTrackingProtection.MainScreen.faviconImage
+
+    var websiteTitle: String {
+        return url.baseDomain ?? ""
+    }
+
+    var connectionStatusString: String {
+        return connectionSecure ?
```

Comment:
curious, why create specific constants for `clearCookiesAlertTitle`, but not for these strings here?

Reply:
the strings are added to specific constants, I used a computed property because it depends on the status of the connection, I'm not sure what you mean

Reply:
small nit, but I was curious on why not change to the following because you were creating constants for the other strings. 
```suggestion
        let secureLabel = .Menu.EnhancedTrackingProtection.connectionSecureLabel
        let unsecureLabel = .Menu.EnhancedTrackingProtection.connectionUnsecureLabel
        return connectionSecure ? secureLabel : unsecureLabel
```